python/paddle/fluid/tests/unittests/test_gather_op.py

```diff
@@ -23,6 +23,21 @@
 from paddle.fluid.dygraph.base import switch_to_static_graph
 
 
+def check_eigh_results(A, eigh_vectors, eigh_values):
```

Comment:
Why do you need to modify the test_gather_op.py here? and why is this test method not placed in the test_eigh_op.py

Reply:
something went wrong. Fixed